From fc766c6db5305effdaaaa843d6a2c2b4623c8b99 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Daniel=20P=2E=20Berrang=C3=A9?= <berrange@redhat.com>
Date: Thu, 29 Jun 2023 17:51:06 +0100
Subject: [PATCH] virt-what-cvm: rename 'azure-hcl' fact to 'hyperv-hcl'
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Azure is a cloud service that uses the HyperV platform, so we
should refer to the fact as 'hyperv-hcl', not 'azure-hcl'.

Signed-off-by: Daniel P. Berrangé <berrange@redhat.com>
(cherry picked from commit 94773022f76f994d7a9b37f59ba978bd28f30d1d)
---
 virt-what-cvm.c   | 2 +-
 virt-what-cvm.pod | 6 +++---
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/virt-what-cvm.c b/virt-what-cvm.c
index 8b8a4df09..52b3426bc 100644
--- a/virt-what-cvm.c
+++ b/virt-what-cvm.c
@@ -214,7 +214,7 @@ cpu_sig_amd (void)
 
     if (cpu_sig_amd_hyperv ()) {
       puts ("amd-sev-snp");
-      puts ("azure-hcl");
+      puts ("hyperv-hcl");
     } else {
       debug("No hyperv CPUID\n");
     }
diff --git a/virt-what-cvm.pod b/virt-what-cvm.pod
index 12cfc6a96..0f9076569 100644
--- a/virt-what-cvm.pod
+++ b/virt-what-cvm.pod
@@ -52,11 +52,11 @@ This is a confidential guest running with Intel TDX technology
 
 Status: tested on Microsoft Azure TDX CVM (preview)
 
-=item B<azure-hcl>
+=item B<hyperv-hcl>
 
 This is a confidential guest running unenlightened under the
-Azure HCL (Host Compatibility Layer). This will be paired with
-B<amd-sev-snp>.
+HyperV (Azure) HCL (Host Compatibility Layer). This will be
+paired with B<amd-sev-snp>.
 
 Status: tested on Microsoft Azure SEV-SNP CVM
 
-- 
2.43.0

